engine/gamesys/src/gamesys/components/comp_gui.cpp

```diff
@@ -2186,6 +2186,37 @@ namespace dmGameSystem
 
         index -= num_properties;
 
+        const char* world_property_names[] = {
```

Comment:
Added new values needed that hold the world space values for the gui nodes. This is needed for Poco to correctly be able to click on an object.